### PR TITLE
Event on connect and keeping the info of the spn connection

### PR DESCRIPTION
**Pull request by vlabo**

### CODE REVIEW

File: captain/navigation.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ findCandidates:
 func connectToHomeHub(ctx context.Context, dst *hub.Hub) error {
 	// Set and clean up exceptions.
 	setExceptions(dst.Info.IPv4, dst.Info.IPv6)
-	defer setExceptions(nil, nil)
+	// defer setExceptions(nil, nil)
```
Comment (dhaavi) [inline captain/navigation.go:113]: If we don't reset the exceptions after connecting, the connections to the node will no go through the SPN. I think this is guarded to only apply to Portmaster itself, so I guess this might never happen anyway.

Was this change on purpose? (As it is only commented out, not removed.)
If so, please elaborate.

File: captain/module.go [hunk 2]
```diff
@@ -21,8 +21,11 @@ import (
 
 var module *modules.Module
 
+const onSPNConnectEvent = "spn connect"
```
Comment (dhaavi) [inline captain/module.go:24]: I would suggest using `spn connected`, as this makes it clear that the connection has been established, and is not in progress anymore.

**Review by dhaavi — CHANGES_REQUESTED** [review]

File: captain/navigation.go (hunk 1 above)
Comment (vlabo) [reply captain/navigation.go:113]: Yes this was intentional. The reason why it is commented is that I didn't understand the reason behind it, will it break something if we don't reset it? It is used only in `checkTunneling` function.

I removed it because after the changes to the firewall now it is possible to run a firewall check to a connection multiple times and change the verdict. If the info for the spn connection is not always present we try to rout the connection through the SPN.
